python/cudf/cudf/pandas/_wrappers/numpy.py

```diff
@@ -105,18 +108,38 @@ def wrap_ndarray(cls, arr: cupy.ndarray | numpy.ndarray, constructor):
         return super(cls, cls)._fsproxy_wrap(arr, constructor)
 
 
+def ndarray__array_ufunc__(self, ufunc, method, *inputs, **kwargs):
+    result, _ = _fast_slow_function_call(
+        getattr(ufunc, method),
+        *inputs,
+        **kwargs,
+    )
+    if isinstance(result, tuple):
```

Comment:
Are there ufuncs that return a tuple of ndarrays?

Reply:
Yes `np.modf`and `np.frexp` for sure. But there are probably others as well